algosdk/transaction.py

```diff
@@ -693,6 +695,31 @@ def address(self):
         addr = encoding.checksum(msig_bytes)
         return encoding.encode_address(addr)
 
+    def verify(self, message):
+        """Verify that the multisig is valid for the message"""
+        try:
+            self.validate()
```

Comment:
does validate() refer to MultisigTransaction.validate()? If so, self.validate() will never work since MultisigTransaction isn't a subclass of Transaction. Maybe this function should go under MultisigTransaction?

Reply:
this is part of Multisig class so that it calls Multisig.validate